src/components/ReportActionItem/IOUAction.js

```diff
@@ -25,11 +29,26 @@ const propTypes = {
     chatReport: PropTypes.shape({
         /** The participants of this report */
         participants: PropTypes.arrayOf(PropTypes.string),
+
+        /** Whether the chat report has an outstanding IOU * */
+        hasOutstandingIOU: PropTypes.bool,
     }),
 
+    /** IOU Report data object */
+    iouReport: PropTypes.shape({
+        /** The currency of the IOUReport */
```

Comment:
I think this is per our naming guidelines.
```suggestion
        /** The currency of the iouReport */
```